lib/utils/workspace-utils.ts

```diff
@@ -124,3 +125,16 @@ export async function deleteWorkspace({ workspaceId, sessionToken }: { workspace
 
   return { data, error };
 }
+
+export const WORKSPACE_ID_COOKIE_NAME = "workspace-id";
```

Comment:
I've made the workspace ID cookie name a constant and I've also renamed it from `workspaceId` to `workspace-id`. The rename should also clear any legacy issues with the cookie for people behind the workspaces feature flag.